### ISSUE TITLE
Limiting # of vehicles saved/Number of Settings windows

### ISSUE THREAD

**Issue by MorroMat** [napaunderground/FuelFinder#29]
Every time you hit the "Add new vehicle" button, it starts a new activity, and there is no limiting how many times you can do it. So, I think that may lead to memory leaks or something.

**Comment by napaunderground** [comment]
Yes, implemented temporary fix.  Need to continue refining.  Working on numVehicles...
